src/services/refactors/convertStringOrTemplateLiteral.ts

```diff
@@ -158,6 +157,11 @@ namespace ts.refactor.convertStringOrTemplateLiteral {
                 indexes.push(index);
                 index++;
             }
+            else if (isNoSubstitutionTemplateLiteral(node)) {
+                text = text + "node";

```

Comment:
I can't tell why `"node"` is being added here.

Reply:
Bah.  This was a semi-debugging leftover: this whole branch should have been removed since `isStringLiteralLike()` covers the `isNoSubstitutionTemplateLiteral()` case.  I now removed it and added a comment about it (and another test too).